plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2038,178 @@ def coupling_parameter(
+
+        T_b : `~astropy.units.Quantity`
+            The temperature of the background field particles.
+
+        Coulomb_log : `~astropy.units.Quantity`
+            The value of the Coulomb logarithm evaluated for the two interacting particles.
+
+        Raises
+        ------
+        `ValueError`
+            If both :math:`v_a` and :math:`T_a` are specified, or neither
+            :math:`v_a` nor :math:`T_a` are specified.
+
+        Notes
+        -----
+        The collision frequencies between two interacting particles are given by four differential equations:
+
+            slowing down - :math:`\frac{d\textbf{v}_{α}}{dt}=-ν_{s}^{α\backslashβ}\textbf{v}_{α}`
+
+            transverse diffusion - :math:`\frac{d}{dt}\left(\overline{\textbf{v}}_{α}-\overline{\textbf{v}}_{α}\right)_{⊥}^{2}=ν_{⊥}^{α\backslashβ}v_{α}^{2}`
+
+            parallel diffusion - :math:`\frac{d}{dt}\left(\overline{\textbf{v}}_{α}-\overline{\textbf{v}}_{α}\right)_{∥}^{2}=ν_{∥}^{α\backslashβ}v_{α}^{2}`
+
+            energy loss - :math:`\frac{d}{dt}v_{α}^{2}=-ν_{ϵ}^{α\backslashβ}v_{α}^{2}`
+
+        For general solutions and limiting cases we encourage the curious reader to refer to p. 31 of :cite:t:`nrlformulary:2019`
+
+        This function uses CGS units under the hood to coincide with our references, but passing MKS units is acceptable.
+
+        Examples
+        --------
+        >>> import astropy.units as u
+        >>> T_a = 1 * u.eV
+        >>> n_b = 1e20 * u.cm**-3
+        >>> T_b = 1e3 * u.eV
+        >>> Coulomb_log = 10 * u.dimensionless_unscaled
+        >>> frequencies = CollisionFrequencies(
+        ...     "e-", "e-", T_a=T_a, n_b=n_b, T_b=T_b, Coulomb_log=Coulomb_log
+        ... )
+        >>> frequencies.slowing_down
+        <Quantity 1.83701432e+11 Hz>
+
+        See Also
+        --------
+        ~plasmapy.formulary.collisions.Coulomb_logarithm : Evaluates the Coulomb
+            logarithm for two interacting electron species.
+
+        ~plasmapy.formulary.collisions.collision_frequency : The Lorentz collision frequency.
+        """
+
+        if v_a is None:
+            if T_a is not None:
+                v_a = thermal_speed(T_a, test_particle).to(u.cm / u.s)
+            else:
+                raise ValueError("Please specify either v_a or T_a.")
+        elif T_a is not None:
+            raise ValueError("Please specify either v_a or T_a, not both.")
+
+        self.test_particle = test_particle
+        self.field_particle = field_particle
+        self.v_a = v_a
+        self.T_a = (
+            T_a
+            if T_a is not None
+            else (0.5 * test_particle.mass.to(u.g) * v_a**2 / k_B).to(u.K)
+        )
+        self.n_b = n_b
+        self.T_b = T_b
+        self.Coulomb_log = Coulomb_log
+
+        x = self._x()
+        phi = self._phi(x)
+        phi_prime = self._phi_prime(x)
+
+        mass_ratio = test_particle.mass / field_particle.mass
+
+        self.v_0 = self.Lorentz_collision_frequency()
```

Comment:
If you want to have this as a shorter alias then you should make this private, e.g. `self._v0`. I don't see a reason to have it around after you calculate the other frequencies though, so maybe just make it a variable rather than an attribute?